lib/plugins/package/lib/zipService.js

```diff
@@ -160,7 +160,13 @@ function excludeNodeDevDependencies(servicePath) {
         return childProcess.execAsync(
```

Comment:
We could use `BbPromise.fromCallback()` here to catch the error stream (as mentioned in the comments).
The callback signature of `childProcess.exec` is `(err, stdin, stderr) => {}`, so it is quite easy to grab it here
and exit the fromCallback properly with an error or success.

Reply:
Definitely will apply this suggestion, thanks.